File: 打家劫舍/C++/original.cpp
class Solution {
public:
    int rob(vector<int>& nums) {
        int now = 0;
        int last = 0;
        for(int i=0; i<nums.size(); ++i){
            int temp = now;
            if(last+nums[i]>now) now = last+nums[i];
            last = temp;
        }
        return now;
    }
};
//思路和python一样

File: 中级算法/排序和搜索/搜索旋转排序数组/python/original.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        '''
        思路：用index函数找到索引，如果没有target，用异常检测返回-1
        '''
        try:
            return nums.index(target)
        except:
            return -1